### ISSUE TITLE
[Bug]: pre-commit --config argument not used?

### ISSUE THREAD

**Issue by HealthyPear** [rstcheck/rstcheck#174]
### To Dos

- [X] I tested with the [latest released version](https://github.com/rstcheck/rstcheck/releases/latest).

- [X] I have checked the [issues](https://github.com/rstcheck/rstcheck/issues) and think that this is not a duplicate.

- [X] I added a very descriptive title to this issue.

### Example Code (rst)

```rst
Not relevant
```

### Relevant log output

```Shell
rstcheck.................................................................Failed
- hook id: rstcheck
- exit code: 2

Usage: rstcheck [OPTIONS] FILES...
Try 'rstcheck --help' for help.
Error No such option: --config . Did you mean --config?
```

### Description

The CLI interface command works as expected, loading info from an `.rstcheck.cfg` file in INI format in the same place as the pre-commit configuration file.

I set up my pre-commit configuration as,

```yaml
  - repo: https://github.com/rstcheck/rstcheck
    rev: v6.1.2
    hooks:
      - id: rstcheck
        args: ["--config ."]
```

### Operating System

macOS

### Operating System Details

macos 11.7.8

working from a conda environment

### Python Version

Python 3.11.3

### rstcheck Version

rstcheck CLI Version: 6.1.2

### Additional Context

 rstcheck-core Version: 1.0.3

pre-commit 3.3.3

**Comment by HealthyPear** [comment]
Sorry, apparently it should be,

```
args: ["--config=."]
```

Though this triggers another issue for me: while the CLI tool doesn't complain, the pre-commit version throws the following multiple errors like this,

```
docs/CONTRIBUTE.rst:31: (INFO/1) No role entry for "ref" in module "docutils.parsers.rst.languages.en".
docs/CONTRIBUTE.rst:31: (ERROR/3) Unknown interpreted text role "ref".
```

so I am not sure if it's working correctly

**Comment by HealthyPear** [comment]
If I add `"--log-level=DEBUG"` to the pre-commit hook arguments I get

```
INFO:rstcheck._cli:Create main configuration from CLI options.
DEBUG:rstcheck._cli:Create main runner instance.
INFO:rstcheck_core.runner:Load config file for main runner: '.rstcheck.cfg'.
DEBUG:rstcheck_core.config:Try loading config file from path: '.rstcheck.cfg'.
DEBUG:rstcheck_core.config:Try loading config file.
DEBUG:rstcheck_core.config:Try loading config from INI file: '/Users/michele/Applications/SWGO/irf-production/.rstcheck.cfg'
DEBUG:rstcheck_core.runner:Merging config from file into main config. File config is dominant: False
DEBUG:rstcheck_core.config:Merging configs.
DEBUG:rstcheck_core.runner:Updating list of files to check.
INFO:rstcheck._cli:Run main runner instance.
INFO:rstcheck_core.runner:Run checks for all files.
DEBUG:rstcheck_core.runner:Runnning checks in parallel with pool size of 4.
```

**Comment by HealthyPear** [comment]
ah shoot I forgot to add also `additional_dependencies: [sphinx]`

this fixes the issue - closing.
